CHANGELOG.md

```diff
@@ -420,6 +425,7 @@ Other minor additions
 * Added new proof to `Data.Fin.Properties`:
   ```agda
   toℕ-fromℕ≤″ : toℕ (fromℕ≤″ m m<n) ≡ m
+  toℕ-cast    : toℕ (cast eq k) ≡ k
```

Comment:
There should be a `toℕ` on the right here?

Reply:
Indeed.